udata/core/user/activities.py

```diff
@@ -70,3 +87,17 @@ def write_activity_on_follow(follow, **kwargs):
             write_activity.delay(
                 UserFollowedUser, current_user._get_current_object(),
                 follow.following)
+
+
+@on_new_discussion.connect
+@on_new_discussion_comment.connect
+def write_activity_on_discuss(discussion, **kwargs):
+    if current_user.is_authenticated:
```

Comment:
I think this is `is_authenticated()` while using Flask-Login 0.2.x. The property is available starting Flask-Login 0.3.x (we need to migrate).
So, it's a false positive as it's always true.